x-pack/qa/multi-project/core-rest-tests-with-multiple-projects/build.gradle

```diff
@@ -47,8 +47,10 @@ tasks.named("yamlRestTest").configure {
   ArrayList<String> blacklist = [
     /* These tests don't work on multi-project yet - we need to go through each of them and make them work */
     '^cat.recovery/*/*',
+    '^cat.repositories/*/*',
     '^cat.snapshots/*/*',
     '^cluster.desired_balance/10_basic/*',
+    '^cluster.stats/10_basic/snapshot stats reported in get cluster stats',
```

Comment:
Re-mute these repository related YAML tests since they require either create, get or delete snapshots to work in MP setup. Previously these tests passed but it was in fact incorrect because they always target the default project.